test/service/test_dss_proxy.py

```diff
@@ -72,7 +72,7 @@ def test_dss_files_proxy(self):
         s3.upload_fileobj(Bucket=bucket_name, Fileobj=io.BytesIO(b'foo'), Key=key)
         file_uuid = '701c9a63-23da-4978-946b-7576b6ad088a'
         file_version = '2018-09-12T121154.054628Z'
-        retry_after = 3
+        retry_after = 1
```

Comment:
This makes the mock DSS return a retry after of 1 which means the test doesn't prove that your changes override the retry after specified by the DSS.
```suggestion
        retry_after = 10
```